codes/universe.py

```diff
@@ -359,3 +359,37 @@ def run(self, problem: ProblemDefinition_Abstract):
             self.converged = MPI.COMM_WORLD.bcast(self.converged, root=0)
         if self.node_number == 0:
             self.postprocess_universe(problem)
+
+class RelativePopulationUniverseDefinition(UniverseDefinition):
```

Comment:
Created a new universe to handle the scoring, which is done relatively instead of absolutely.